### PR TITLE
added the Desktop version for my Portfolio

### PR DESCRIPTION
**Pull request by Cluab**
completed adding the desktop version for my first Portfolio in Microverse

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,117 +16,133 @@
...
           
-       </div>
-       <div class="worktwo">
-        <h5 class="work-two-title">Profesional Art Printing Data</h5>
-        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
-          or sign-ups required. has been the industry's standard</p>
-          <div class="see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
+       <div class="workthree worktwo">
+        <div class="work-box-three work-box-two">
+        <h5 class="work-three-title work-two-title">Data Dashboard Healthcare</h5>
```
Comment (nedjwab) [inline index.html:85]: [ ] kindly add a hover effect to your cards, so when we hover on with the mouse the description and dark opacity disappear and a See Project button appears at the bottom of the image and takes the whole image width mentioned button should take you to the Live Demo of the project or to the GitHub repository.👀

File: index.html [hunk 2]
```diff
@@ -16,117 +16,133 @@
...
+          <h5 class="work-three-title work-two-title">Website Protfolio</h5>
+          <p class="work-three-dis work-two-dis">A daily selection of privately personalized reads; no accounts 
+            or sign-ups required. has been the industry's standard</p>
+            <ul class="languages-work">
+              <li class="html-box"><p class="code-name">html</li>
+              <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+              <li class="ruby-box"><p class="code-name">Ruby</p></li>
+            </ul>
+          </div>
+            <div class="see-three-box see-two-box"><button class="see-two"  href="index.html">see Project</button></div>
```
Comment (nedjwab) [inline index.html:144]: [ ] kindly add a hover effect to all your buttons as it's mentioned on the Figma template 🤓.
Your buttons 👀:  
hisbutton


the Figma buttons 👀:  
hoverbuuton

File: index.html [hunk 3]
```diff
@@ -192,10 +211,11 @@ <h2 class="aboutmeintro">About me</h2>
     </section>
     
       <div class="footer">
-      <form class="form"action="https://formspree.io/f/mjvzwzlb" method="post">
         <h2 class="footer-intro">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
```
Comment (nedjwab) [inline index.html:214]: [ ] kindly fix the text position in the desktop version, so it looks like the Figma design 🤓.
your text : 
histext

The Figma design text : 
textdesign

File: styles.css [hunk 4]
```diff
@@ -659,11 +689,490 @@ textarea {
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    font-size: 14px;
+    line-height: 24px;
+    margin-left: 0;
+  }
+
+  .diffrent-work {
+    display: grid;
```
Comment (nedjwab) [inline styles.css:944]: Good implementation of the grid 🔥✔

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Cluab  

Nice job✔, I liked your implementation 🔥, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Good design 🔥🔥🔥 , keep it up.😉

## (Changes Required) Aspects to improve: ♻️
You did great implementing  the project requirements 🔥🔥, but you still have some issues:  
Check the comments below 👇
  
## [Optional] suggestions:

-  Nothing to mention. 

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (Cluab) [reply index.html:85]: hello Nedjwab the dark opacity is engraved in the card image and in the project requirements nothing of that sort is written in it  https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md

Comment (Cluab) [reply index.html:85]: you can check the link

Comment (nedjwab) [reply index.html:85]: @Cluab , it's showing in the design, this is the hover effect :
hoverdesign
your card without the hover effect should look like this : 

without

**Review by ichala — APPROVED** [review]
Hi @Cluab  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes implemted ✔️ 

 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


*************************************
# Optional Suggestions 

- N/A




_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**